generate-readme.sc

```diff
@@ -0,0 +1,34 @@
+import ammonite.ops._
+
+val appsPaths = ls! pwd/'apps/'resources
+val contribPaths = ls! pwd/"apps-contrib"/'resources
+
+val appNames = appsPaths.map(_.baseName).toList
+val contribNames = contribPaths.map(_.baseName).toList
+
+
+val readme = s"""|# Coursier Apps
+                 |
+                 |[![Build Status](https://travis-ci.org/coursier/apps.svg?branch=master)](https://travis-ci.org/coursier/apps)
+                 |[![Maven Central](https://img.shields.io/maven-central/v/io.get-coursier/apps.svg)](https://maven-badges.herokuapp.com/maven-central/io.get-coursier/apps)
+                 |
+                 |This repository holds the apps in the Main and Contrib channels for Coursier. You can find information about
+                 |creating your own application to be installed with `cs` [here on the website](https://get-coursier.io/docs/cli-install.html#creating-your-own-applications).
+                 |
+                 |## Main
+                 |These are the apps in the default main JAR-based channel, `io.get-coursier:app` which is used with `cs install`
+                 |
+                 |${appNames.mkString(" - ", "\n - ", "")}
+                 |
+                 |## Contrib
+                 |These apps are available by passing `--contrib` to the `cs install` command.
+                 |Feel free to send in a PR to add your application here!
+                 |
+                 |${contribNames.mkString(" - ", "\n - ", "")}
+                 |
+                 |### Updating the readme
+                 |This readme is auto-generated by the `generate-readme.sc` Ammonite script in the root of this directory. Please don't
+                 |update the readme directy, but rather update the script.
+                 |""".stripMargin
```

Comment:
nit: maybe we can keep the README template in a separate file, like `README.md.template`, so that it's easier to edit.

Reply:
Sure I'll add in a template file.